### DISCUSSION TITLE
New Environment가 생성되지 않습니다.

### DISCUSSION THREAD

**Discussion by gwililgwi** [naverz/zepeto-studio-kor#571] [5. Q&A]
Invalid generated unique path '' (input path 'Assets/Assets/New Environment.asset')
UnityEditor.ProjectWindowUtil:CreateAsset (UnityEngine.Object,string)
ZEPETO.World.Editor.WorldToolbar/<>c:<BuildOptions>b__18_1 ()
UnityEditor.GenericMenu:CatchMenu (object,string[],int)

퍼블리시 옆의 버튼을 통해 create custom enviroment를 클릭하면
위와 같은 오류가 뜨면서 생성이 되지 않습니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 이슈 재현이 되지 않아서 개발 환경 및 월드 패키지 버전도 같이 기재 부탁 드립니다.
아래 문의 가이드를 참고하여 주세요.
https://github.com/naverz/zepeto-studio-kor/discussions/546

**Reply by gwililgwi** [reply]
> 안녕하세요, 해당 이슈 재현이 되지 않아서 개발 환경 및 월드 패키지 버전도 같이 기재 부탁 드립니다. 아래 문의 가이드를 참고하여 주세요. #546

네 알겠습니다. 다시 해보고 수정해서 올리도록 하겠습니다!
